apps/editor/src/js/utils/dom.js

```diff
@@ -1234,7 +1234,7 @@ function getOffset(element) {
   while (element !== document.body) {
     top += element.offsetTop;
     left += element.offsetLeft;
-    element = element.offsetParent;
+    element = element.offsetParent || element.parentElement;
   }
```

Comment:
I think it can be solved by modifying with the `do-while` statement as below. Can you check it?

```js
do {
  top += element.offsetTop || 0;
  left += element.offsetLeft || 0;
  element = element.offsetParent;
} while (element);
```

Reply:
Sure it works :)